<ui version="4.0" >
 <class>BitsGUI</class>
 <widget class="QMainWindow" name="BitsGUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" native="1" name="hotkeyHolder" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>0</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="mainLayout" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="alternatingRowColors" >
        <bool>false</bool>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="dataViewSplitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title" >
         <string>Job Properties</string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="2" >
            <widget class="QTimeEdit" name="jobStartTime" >
             <property name="calendarPopup" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="estEndTTime" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Estimated End Time:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2" >
            <widget class="QLineEdit" name="jobFilename" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>c:\Program Files\TaskScheduler\job1.txt</string>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2" >
            <widget class="QTimeEdit" name="jobEndTime" />
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Configuration File:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="jobNameTextBox" />
           </item>
           <item row="2" column="1" >
            <widget class="QDateEdit" name="jobEndDate" >
             <property name="displayFormat" >
              <string>MMMM d, yyyy</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="jobNameTLabel" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>0</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Job Name:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QDateEdit" name="jobStartDate" >
             <property name="displayFormat" >
              <string>MMMM d, yyyy</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Estimated Start Time:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QPushButton" name="generateReport" >
             <property name="styleSheet" >
              <string/>
             </property>
             <property name="text" >
              <string>  Generate Job Report</string>
             </property>
             <property name="icon" >
              <iconset>Icons/format-justify-center.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="3" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>75</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="taskProperties" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Task Properties</string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Estimated Start Time:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item rowspan="2" row="3" column="1" colspan="2" >
            <widget class="QLineEdit" name="taskTotalTime" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Estimated End Time:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" >
            <widget class="QTimeEdit" name="taskEndTime" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="taskName" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>0</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Task Name:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2" >
            <widget class="QTableView" name="attributeView" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>7</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editTriggers" >
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::NoEditTriggers|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="showDropIndicator" stdset="0" >
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
             <property name="selectionMode" >
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior" >
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QDateEdit" name="taskEndDate" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="displayFormat" >
              <string>MMMM d, yyyy</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item rowspan="2" row="3" column="3" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>75</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2" >
            <widget class="QTimeEdit" name="taskStartTime" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="7" column="0" >
            <widget class="QLabel" name="label_13" >
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Other Attributes:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item rowspan="2" row="3" column="0" >
            <widget class="QLabel" name="label_6" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>5</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Total Time Spent:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2" >
            <widget class="QWidget" native="1" name="widget" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>9</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QPushButton" name="startWorkButton" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Start Work</string>
                </property>
                <property name="icon" >
                 <iconset>Icons/go-next.png</iconset>
                </property>
                <property name="iconSize" >
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="shortcut" >
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopWorkButton" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Stop Work</string>
                </property>
                <property name="icon" >
                 <iconset>Icons/process-stop.png</iconset>
                </property>
                <property name="iconSize" >
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="setHotKey" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Set Hot Key</string>
                </property>
                <property name="icon" >
                 <iconset>Icons/list-add.png</iconset>
                </property>
                <property name="iconSize" >
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QDateEdit" name="taskStartDate" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>7</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="displayFormat" >
              <string>MMMM d, yyyy</string>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Task Progress:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2" >
            <widget class="QWidget" native="1" name="widget_2" >
             <widget class="QRadioButton" name="incompleteRadio" >
              <property name="geometry" >
               <rect>
                <x>55</x>
                <y>-5</y>
                <width>92</width>
                <height>22</height>
               </rect>
              </property>
              <property name="text" >
               <string>Incomplete</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="completeRadio" >
              <property name="geometry" >
               <rect>
                <x>195</x>
                <y>-5</y>
                <width>92</width>
                <height>22</height>
               </rect>
              </property>
              <property name="text" >
               <string>Complete</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="newJobAction" />
    <addaction name="separator" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="newJobAction" />
  </widget>
  <action name="newJobAction" >
   <property name="icon" >
    <iconset>Icons/accessories-text-editor.png</iconset>
   </property>
   <property name="text" >
    <string>New Job</string>
   </property>
   <property name="iconText" >
    <string>New Job</string>
   </property>
   <property name="toolTip" >
    <string>Create a new job</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
